scripts/versions.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
 # Don't export them as they're used in the context of other calls
-avalanche_version=${AVALANCHE_VERSION:-'v1.10.14'}
+avalanche_version=${AVALANCHE_VERSION:-'826f9415'}
```

Comment:
can we tag avalanchego with an rc and use it here?

Reply:
We didn't use an `rc` here since there were none tagged. We should be able to test against commits without rcs (@marun shipped this in #366)

Reply:
Does this work? https://github.com/ava-labs/avalanchego/releases/tag/v1.10.15-rc.1

Reply:
Does this work? https://github.com/ava-labs/avalanchego/releases/tag/v1.10.15-rc.1